test/fixtures/sql/fikspunkt_forvaltning.sql

```diff
@@ -1,48 +1,49 @@
 CREATE TABLE BEREGNING (
+
    OBJECTID INTEGER GENERATED ALWAYS AS IDENTITY (START WITH 1 INCREMENT BY 1 ORDER NOCACHE) PRIMARY KEY,
    REGISTRERINGFRA TIMESTAMP WITH TIME ZONE NOT NULL,
    REGISTRERINGTIL TIMESTAMP WITH TIME ZONE,
-   SAGSEVENTID VARCHAR2(4000) NOT NULL
+   SAGSEVENTID VARCHAR2(36) NOT NULL
```

Comment:
I realize that `VARCHAR(4000)` is a bit excessive in most places here, but why change it here? Seems like something that should be fixed upstream.